functions/Get-DbaAgentJobOutputFile.ps1

```diff
@@ -135,17 +134,13 @@ param
                     }
                     else
                     {
-                        Write-Verbose "$(($Step).Name) for $JobName has No Output File"
+                        Write-Verbose "$(($Step).Name) for $($JobName) has No Output File"
```

Comment:
this is the same, $jobname is the submitted job, if there is one
$job.Name should contain the jobname for the job in the loop